idaes/power_generation/carbon_capture/mea_solvent_system/unit_models/tests/test_column.py

```diff
@@ -0,0 +1,297 @@
+                "property_package": m.fs.vapor_properties,
+                "has_pressure_change": False,
+                "pressure_drop_type": None},
+            "liquid_side":
+            {
+                "transformation_scheme": "FORWARD",
+                "property_package": m.fs.liquid_properties
+            }})
+
+        # Time discretization
+        discretizer = TransformationFactory('dae.finite_difference')
+        discretizer.apply_to(m.fs, wrt=m.fs.time, nfe=t_nfe, scheme='BACKWARD')
+
+        # Fix inputs variables
+        m.fs.col.diameter_column.fix(0.64135)
+        if Fpar == ProcessType.absorber:
+            m.fs.col.length_column.fix(18.15)
+            # Rich loading at the bottom of column @ final time
+            m.fs.col.loading = Param(initialize=0.4927155969073804)
+            for t in m.fs.time:
+                # Vapor
+                m.fs.col.vap_in_flow[t].fix(21.48)
+                m.fs.col.vap_in_temperature[t].fix(317.88)
+                m.fs.col.pressure_bottom[t].fix(107650)
+                m.fs.col.vap_in_mole_frac[t, "CO2"].fix(0.11453)
+                m.fs.col.vap_in_mole_frac[t, "H2O"].fix(0.08526)
+                m.fs.col.vap_in_mole_frac[t, "N2"].fix(0.73821)
+                m.fs.col.vap_in_mole_frac[t, "O2"].fix(0.06200)
+                # Liquid
+                m.fs.col.liq_in_flow[t].fix(37.55)
+                m.fs.col.liq_in_temperature[t].fix(319.87)
+                m.fs.col.liq_in_mole_frac[t, "CO2"].fix(0.00963)
+                m.fs.col.liq_in_mole_frac[t, "H2O"].fix(0.87435)
+                m.fs.col.liq_in_mole_frac[t, "MEA"].fix(0.11602)
+
+            # Initialize column
+            m.fs.col.initialize()
+        elif Fpar == ProcessType.stripper:
+            m.fs.col.length_column.fix(12.1)
+            # Lean loading at the bottom of column @ final time
+            m.fs.col.loading = Param(initialize=0.17982818165156983)
+            for t in m.fs.time:
+                # Vapor
+                m.fs.col.vap_in_flow[t].fix(17.496)
+                m.fs.col.vap_in_temperature[t].fix(396.6)
+                m.fs.col.pressure_bottom[t].fix(183430)
+                m.fs.col.vap_in_mole_frac[t, "CO2"].fix(0.0145)
+                m.fs.col.vap_in_mole_frac[t, "H2O"].fix(0.9855)
+                # Liquid
+                m.fs.col.liq_in_flow[t].fix(84.48)
+                m.fs.col.liq_in_temperature[t].fix(382.15)
+                m.fs.col.liq_in_mole_frac[t, "CO2"].fix(0.0331)
+                m.fs.col.liq_in_mole_frac[t, "H2O"].fix(0.8547)
+                m.fs.col.liq_in_mole_frac[t, "MEA"].fix(0.1122)
+
+            # Initialize column
+            m.fs.col.initialize(
+                homotopy_steps_h=[0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1])
+        return m
+
+# ------------------------------------------------------------------------------
+    @pytest.mark.skipif(solver is None, reason="Solver not available")
+    @pytest.mark.integration
```

Comment:
These tests should to be `unit` or `component`  tests.

The general rules are:

1. `unit` tests should run in under 2 seconds and cannot involve a solver.
2.  `component` tests can be longer (but ideally less than 10 seconds) and can use a solver.
3. `integration` tests are for long tests, such as dynamic examples or verification/validation tests against a large set of data.

For unit model, most tests should be `unit` or `component` tests, as we want to run these frequently to make sure everything works.